R/get.R

```diff
@@ -63,6 +67,8 @@ get_constant_vars <- function(dataset, by_vars, ignore_vars = NULL) {
 #'
 #' get_duplicates(c("a", "a", "b", "c", "d", "d"))
 get_duplicates <- function(x) {
+  assert_that(is.atomic(x), msg = "x must be an atomic vector")
```

Comment:
We are trying to get away from using `assert_that` functions.  Could we use something else to replace it? https://pharmaverse.github.io/admiraldev/devel/reference/index.html

Reply:
Hi Ben, thank you for letting me know. Is it ok if I add a function, to check if input is an atomic vector, into assertions.R?

Reply:
ooohh like you want to make a new `assert` function?  Please do!!

Reply:
Also, I think okay to make in the same branch.  Since it has common theme

Reply:
I added the function as suggested :) Let me know if you have any suggestions Ben, thank you!